lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -111,7 +111,7 @@ def define
         end
 
         next_node do
-          if %w(italy sweden).include?(calculator.ceremony_country)
+          if %w(italy sweden mozambique).include?(calculator.ceremony_country)
```

Comment:
It might be a good moment to extract a method to store the list of flattened countries:

```ruby
if calculator.content_flattened?
```